apiServer/pxone/utils/k8sutils.go

```diff
@@ -57,3 +58,55 @@ func ExecuteHelmCmd(c *gin.Context) {
 		"message": "Command received and executed successfully",
 	})
 }
+
+func GetNamespaceID(c *gin.Context) {
+	ns := c.Param("namespace")
+	namespace, err := k8sCore.GetNamespace(ns)
+	if err != nil {
+		c.JSON(http.StatusInternalServerError, gin.H{
+			"error": err.Error(),
+		})
+		return
+	}
+	clusterID := string(namespace.GetObjectMeta().GetUID())
+	c.JSON(http.StatusOK, gin.H{
+		"clusterID": clusterID,
```

Comment:
it is returning clusterID but func says GetNamespaceID. is this correct?

Reply:
Yes, cluster ID is kube-system namespace ID